internal/provider/resource_user.go

```diff
@@ -88,6 +94,28 @@ func resourceUserSchemaToModel(d *schema.ResourceData, c *APIClient) (*api.UserR
 	return &m, nil
 }
 
+func resourceUserSetPassword(d *schema.ResourceData, c *APIClient, ctx context.Context) diag.Diagnostics {
+	password, ok := d.Get("password").(string)
+	if !ok || password == "" {
+		return nil
+	}
+	if !d.HasChange("password") {
```

Comment:
if you're checking that here, you also have to add a `|| d.IsNewResource()` I think

Reply:
I meant `&& !d.IsNewResource()`. `!d.HasChange("password")` will evaluate to true on resource creation, and thus the password won't be set